src/common/hashtable/src/traits.rs

```diff
@@ -470,7 +507,18 @@ pub trait HashJoinHashtableLike {
     type Key: ?Sized;
 
     // Using hashes to probe hash table and converting them in-place to pointers for memory reuse.
-    fn probe(&self, hashes: &mut [u64]);
+    fn probe(&self, hashes: &mut [u64], bitmap: Option<Bitmap>) -> usize;
+
+    // Using hashes to probe hash table and converting them in-place to pointers for memory reuse.
+    fn early_filtering_probe(&self, hashes: &mut [u64], bitmap: Option<Bitmap>) -> usize;
+
+    // Using hashes to probe hash table and converting them in-place to pointers for memory reuse.
```

Comment:
We can make the notes more meaningful to distinguish the three methods;

Reply:
Done.